src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,10 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
+        PreferencesManager.on("change", "uppercaseColors", function () {
+            this._isUpperCase = PreferencesManager.get("uppercaseColors");
+        }.bind(this));
```

Comment:
Drive-by comment: this leaks listeners.  Each time a color editor is opened, another change listener is added to PreferencesManager.  The change listener will also keep the entire color editor in memory forever, since it's part of its closure.  We shouldn't merge until this is fixed.